### PR TITLE
hotfix: revert to corejs@2

### PR DESCRIPTION
**Pull request by pi0**
Revert core-js to v2 as it is the default for current babel and forcing to v3 causes issues like #5313. Upgrading is also a breaking change for projects that have a babel.config file. So it makes sense to do it for nuxt3 or when ecosystem adopted more to v3.

### CODE REVIEW

File: packages/babel-preset-app/src/index.js [hunk 1]
```diff
@@ -1,19 +1,19 @@
...
-  'es.promise.finally'
+  'es6.object.assign',
+  // #2012 es7.promise replaces native Promise in FF and causes missing finally
+  'es7.promise.finally'
 ]
 
 function getPolyfills(targets, includes, { ignoreBrowserslistConfig, configPath }) {
   const { isPluginRequired } = require('@babel/preset-env')
-  const builtInsList = require('core-js-compat/data')
+  const builtInsList = require('@babel/preset-env/data/built-ins.json.js')
```
Comment (clarkdo) [inline packages/babel-preset-app/src/index.js]: Shoule be json?

**Comment by clarkdo** [comment]
LGTM

**Comment by clarkdo** [comment]
One thing need to be noticed, if any lib has already updated to core-js 3, this is still an issue.

**Comment by simllll** [comment]
just ran into this issue, we adopted already everything to core js 3.. actually I was just waiting for 2.5 to come because of core js 3. Any other idea how we could solve this?

**Comment by pi0** [comment]
@simllll sorry to hear that :( Actually we've fased lots of problems because of libs that _implicitly_ depend on corejs 2 and we can't support both at a same time. How did you get prepared for 3.x?

**Comment by simllll** [comment]
Yeah we had similar troubles, but switched or patched some libraries to get over it. Wasn't that much effort in the end for us, but still not so cool if we would revert this now.

I just read clarkdos comment that he is working on a option/flag to set the core is version, this would solve our problem! So I just fixed our version to 2.5.0 for now.

**Comment by pi0** [comment]
@simllll You requested and it will be possible with next version (2.6.0) via #5411. All thanks to @clarkdo :heart:

**Comment by simllll** [comment]
Very nice :) thanks guys for this awesome project!

**Pull request merged**